src/web/components/elements/QuizBlockComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { space } from '@guardian/src-foundations';
+
+import {
+    PersonalityQuizAtom,
+    KnowledgeQuizAtom,
+} from '@guardian/atoms-rendering';
+
+type Props = {
+    id: string;
+    questions: QuestionType[];
+    resultBuckets?: ResultBucketsType[];
+    quizType: 'personality' | 'knowledge';
+};
+export const QuizBlockComponent = ({
+    id,
+    questions,
+    resultBuckets,
+    quizType,
+}: Props) => (
+    <div
+        className={css`
+            padding-bottom: ${space[2]}px;
+        `}
+    >
```

Comment:
This div can be deleted in favour of Figure in `ArticleRenderer`

Reply:
removed in favor of `<Figure>`